k8_cortx_cloud/destroy-cortx-cloud.sh

```diff
@@ -198,7 +198,7 @@ function deleteCortxPVs()
         if [[ "${force_delete}" == "--force" || "${force_delete}" == "-f" ]]; then
             kubectl patch pv "${persistent_volume}" -p '{"metadata":{"finalizers":null}}'
         fi
-        kubectl delete pv "${persistent_volume}"
+        kubectl delete pv "${persistent_volume}" --ignore-not-found
```

Comment:
I don't think it's really necessary to manually delete PVs anymore, but I've left it in. I added these ignore flags since there are race conditions (provisioner deletes PV first) that produce "not found" errors from kubectl. Since the script doesn't check for errors, it sort of pointless, but seemed good to do.

Reply:
This has been removed.